neonvm/hack/Dockerfile.kernel-builder

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:23.10 AS build-deps
 
-ARG KERNEL_VERSION
+ARG KERNEL_VERSION=6.1.63
```

Comment:
Is it required to re-generate linux-config for each kernel update, major update? If so, then it'll be convenient to keep linux-config file version-suffixed, just to avoid possible missynchronisation.

Reply:
Ah yeah, makes sense. I'll keep it version-suffixed.